src/content/eth2/merge/index.md

```diff
@@ -4,29 +4,29 @@ description: Learn about the merge - when mainnet Ethereum joins the Beacon Chai
+    "This will mark the end of proof-of-work for Ethereum, and the full transition to proof-of-stake.",
+    "This is planned to preceed the roll out of shard chains.",
+    'We formerly referred to this as "the docking."',
   ]
 ---
 
 <UpgradeStatus date="~2021/22">
-    This upgrade will follow the arrival of shard chains. But it’s the moment where the <a href="/eth2/vision/">Eth2 vision</a> becomes fully realised – more scalability, security, and sustainability with staking supporting the whole network.
+    This upgrade is currently planned to follow the London upgrade, which is coming in the summer of 2021. It represents the disabling of proof-of-work mining, and upgrades the Ethereum you know and love today to a network secured by staked ether. An exciting step in realizing the <a href="/eth2/vision/">Eth2 vision</a> – more scalability, security, and sustainability.
 </UpgradeStatus>
 
 ## What is the merge? {#what-is-the-docking}
 
-It's important to remember that initially, the other Eth2 upgrades are being shipped separately from [mainnet](/glossary/#mainnet) - the chain we use today. Ethereum mainnet will continue to be secured by [proof-of-work](/developers/docs/consensus-mechanisms/pow/), even while [the Beacon Chain](/eth2/beacon-chain/) and its [shard chains](/eth2/shard-chains/) run in parallel using [proof of stake](/developers/docs/consensus-mechanisms/pos/). The merge is when these two systems dock together.
+It's important to remember that initially, the other Eth2 upgrades are being shipped separately from [mainnet](/glossary/#mainnet) - the chain we use today. Ethereum mainnet continues to be secured by [proof-of-work](/developers/docs/consensus-mechanisms/pow/), even while [the Beacon Chain](/eth2/beacon-chain/) runs in parallel using [proof-of-stake](/developers/docs/consensus-mechanisms/pos/). The merge is when these two systems finally come together.
 
-Imagine Ethereum is a space ship that isn’t quite ready for an interstellar voyage. With the Beacon Chain and the shard chains the community has built a new engine and a hardened hull. When it’s time, the current ship will dock with this new system, merging into one ship, ready to put in some serious lightyears and take on the universe.
+Imagine Ethereum is a space ship that isn’t quite ready for an interstellar voyage. With the Beacon Chain the community has built a new engine and a hardened hull. When it’s time, the current ship will dock with this new system, merging into one ship, ready to put in some serious lightyears and take on the universe.
 
 ## Merging with mainnet {#docking-mainnet}
 
-When ready, Ethereum mainnet will "merge" with the Beacon Chain, becoming its own shard which uses proof-of-stake instead of [proof of work](/developers/docs/consensus-mechanisms/pow/).
+When ready, Ethereum mainnet will "merge" with the Beacon Chain, becoming its own shard which uses proof-of-stake instead of [proof-of-work](/developers/docs/consensus-mechanisms/pow/).
```

Comment:
I don't think we need to hyphenate proof-of-work or proof-of-stake if they fall at the end of a sentence. See the rules here: https://www.grammarly.com/blog/hyphen/

Reply:
this change is still outstanding